src/conways.rs

```diff
@@ -0,0 +1,126 @@
+const COLUMNS: usize = 32;
+const ROWS: usize = 32;
+
+#[derive(Clone, Copy, PartialEq, Debug)]
+pub enum CellState {
+    Alive,
+    Dead,
+}
+
+#[derive(Clone, Copy)]
+pub struct Position(pub usize, pub usize);
```

Comment:
I think it might be simpler to use a type rename here:
`type Position = (usize, usize)`

Reply:
Done!